[Wed Apr 20 12:28:18.651678 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 12:28:18.651714 2022] [core:notice] [pid 1] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 12:31:06.778934 2022] [mpm_prefork:notice] [pid 1] AH00169: caught SIGTERM, shutting down
[Wed Apr 20 12:31:17.291737 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 12:31:17.291766 2022] [core:notice] [pid 1] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 12:32:27.642072 2022] [mpm_prefork:notice] [pid 1] AH00169: caught SIGTERM, shutting down
[Wed Apr 20 12:33:38.709556 2022] [mpm_prefork:notice] [pid 1] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 12:33:38.709587 2022] [core:notice] [pid 1] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 12:34:14.723183 2022] [mpm_prefork:notice] [pid 1] AH00169: caught SIGTERM, shutting down
[Wed Apr 20 12:42:17.878566 2022] [mpm_prefork:notice] [pid 860] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 12:42:17.878600 2022] [core:notice] [pid 860] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 13:03:42.932667 2022] [mpm_prefork:notice] [pid 858] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 13:03:42.933158 2022] [core:notice] [pid 858] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 13:14:20.746567 2022] [mpm_prefork:notice] [pid 860] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 13:14:20.747115 2022] [core:notice] [pid 860] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 15:52:11.814766 2022] [mpm_prefork:notice] [pid 868] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 15:52:11.815264 2022] [core:notice] [pid 868] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 20 17:28:10.088259 2022] [mpm_prefork:notice] [pid 854] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 20 17:28:10.088697 2022] [core:notice] [pid 854] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-20 17:33:16] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 09:42:27.709416 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 09:42:27.709816 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 09:47:31] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  SELECT (CASE WHEN count(*) > 100 THEN 1 ELSE 0 END) AS "over_threshold"  FROM "ElementalArea_Live"  Table 'SS_mysite.ElementalArea_Live' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nSELECT (CASE WHEN count(*) > 100 THEN 1 ELSE 0 END) AS \"over_threshold\"\n\nFROM \"ElementalArea_Live\"\n\nTable 'SS_mysite.ElementalArea_Live' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-21 09:49:07] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  SELECT DISTINCT 'Virtual page' AS "DependentLinkType", "SiteTree"."ClassName", "SiteTree"."LastEdited", "SiteTree"."Created", "SiteTree"."CanViewType", "SiteTree"."CanEditType", "SiteTree"."Version", "SiteTree"."URLSegment", "SiteTree"."Title", "SiteTree"."MenuTitle", "SiteTree"."Content", "SiteTree"."MetaDescription", "SiteTree"."ExtraMeta", "SiteTree"."ShowInMenus", "SiteTree"."ShowInSearch", "SiteTree"."Sort", "SiteTree"."HasBrokenFile", "SiteTree"."HasBrokenLink", "SiteTree"."ReportClass", "SiteTree"."ParentID", "Page"."ElementalAreaID", "VirtualPage"."VersionID", "VirtualPage"."CopyContentFromID", "SiteTree"."ID",  			CASE WHEN "SiteTree"."ClassName" IS NOT NULL THEN "SiteTree"."ClassName" 			ELSE 'SilverStripe\\CMS\\Model\\SiteTree' END AS "RecordClassName"  FROM "SiteTree" LEFT JOIN "Page" ON "Page"."ID" = "SiteTree"."ID" LEFT JOIN "VirtualPage" ON "VirtualPage"."ID" = "SiteTree"."ID"  WHERE ("VirtualPage"."CopyContentFromID" = ?)  AND ("SiteTree"."ClassName" IN (?))  ORDER BY "SiteTree"."Sort" ASC  Table 'SS_mysite.Page' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nSELECT DISTINCT 'Virtual page' AS \"DependentLinkType\", \"SiteTree\".\"ClassName\", \"SiteTree\".\"LastEdited\", \"SiteTree\".\"Created\", \"SiteTree\".\"CanViewType\", \"SiteTree\".\"CanEditType\", \"SiteTree\".\"Version\", \"SiteTree\".\"URLSegment\", \"SiteTree\".\"Title\", \"SiteTree\".\"MenuTitle\", \"SiteTree\".\"Content\", \"SiteTree\".\"MetaDescription\", \"SiteTree\".\"ExtraMeta\", \"SiteTree\".\"ShowInMenus\", \"SiteTree\".\"ShowInSearch\", \"SiteTree\".\"Sort\", \"SiteTree\".\"HasBrokenFile\", \"SiteTree\".\"HasBrokenLink\", \"SiteTree\".\"ReportClass\", \"SiteTree\".\"ParentID\", \"Page\".\"ElementalAreaID\", \"VirtualPage\".\"VersionID\", \"VirtualPage\".\"CopyContentFromID\", \"SiteTree\".\"ID\", \n\t\t\tCASE WHEN \"SiteTree\".\"ClassName\" IS NOT NULL THEN \"SiteTree\".\"ClassName\"\n\t\t\tELSE 'SilverStripe\\\\CMS\\\\Model\\\\SiteTree' END AS \"RecordClassName\"\n\nFROM \"SiteTree\"\nLEFT JOIN \"Page\" ON \"Page\".\"ID\" = \"SiteTree\".\"ID\"\nLEFT JOIN \"VirtualPage\" ON \"VirtualPage\".\"ID\" = \"SiteTree\".\"ID\"\n\nWHERE (\"VirtualPage\".\"CopyContentFromID\" = ?)\n AND (\"SiteTree\".\"ClassName\" IN (?))\n\nORDER BY \"SiteTree\".\"Sort\" ASC\n\nTable 'SS_mysite.Page' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-21 09:49:09] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  SELECT DISTINCT 'Virtual page' AS "DependentLinkType", "SiteTree"."ClassName", "SiteTree"."LastEdited", "SiteTree"."Created", "SiteTree"."CanViewType", "SiteTree"."CanEditType", "SiteTree"."Version", "SiteTree"."URLSegment", "SiteTree"."Title", "SiteTree"."MenuTitle", "SiteTree"."Content", "SiteTree"."MetaDescription", "SiteTree"."ExtraMeta", "SiteTree"."ShowInMenus", "SiteTree"."ShowInSearch", "SiteTree"."Sort", "SiteTree"."HasBrokenFile", "SiteTree"."HasBrokenLink", "SiteTree"."ReportClass", "SiteTree"."ParentID", "Page"."ElementalAreaID", "VirtualPage"."VersionID", "VirtualPage"."CopyContentFromID", "SiteTree"."ID",  			CASE WHEN "SiteTree"."ClassName" IS NOT NULL THEN "SiteTree"."ClassName" 			ELSE 'SilverStripe\\CMS\\Model\\SiteTree' END AS "RecordClassName"  FROM "SiteTree" LEFT JOIN "Page" ON "Page"."ID" = "SiteTree"."ID" LEFT JOIN "VirtualPage" ON "VirtualPage"."ID" = "SiteTree"."ID"  WHERE ("VirtualPage"."CopyContentFromID" = ?)  AND ("SiteTree"."ClassName" IN (?))  ORDER BY "SiteTree"."Sort" ASC  Table 'SS_mysite.Page' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nSELECT DISTINCT 'Virtual page' AS \"DependentLinkType\", \"SiteTree\".\"ClassName\", \"SiteTree\".\"LastEdited\", \"SiteTree\".\"Created\", \"SiteTree\".\"CanViewType\", \"SiteTree\".\"CanEditType\", \"SiteTree\".\"Version\", \"SiteTree\".\"URLSegment\", \"SiteTree\".\"Title\", \"SiteTree\".\"MenuTitle\", \"SiteTree\".\"Content\", \"SiteTree\".\"MetaDescription\", \"SiteTree\".\"ExtraMeta\", \"SiteTree\".\"ShowInMenus\", \"SiteTree\".\"ShowInSearch\", \"SiteTree\".\"Sort\", \"SiteTree\".\"HasBrokenFile\", \"SiteTree\".\"HasBrokenLink\", \"SiteTree\".\"ReportClass\", \"SiteTree\".\"ParentID\", \"Page\".\"ElementalAreaID\", \"VirtualPage\".\"VersionID\", \"VirtualPage\".\"CopyContentFromID\", \"SiteTree\".\"ID\", \n\t\t\tCASE WHEN \"SiteTree\".\"ClassName\" IS NOT NULL THEN \"SiteTree\".\"ClassName\"\n\t\t\tELSE 'SilverStripe\\\\CMS\\\\Model\\\\SiteTree' END AS \"RecordClassName\"\n\nFROM \"SiteTree\"\nLEFT JOIN \"Page\" ON \"Page\".\"ID\" = \"SiteTree\".\"ID\"\nLEFT JOIN \"VirtualPage\" ON \"VirtualPage\".\"ID\" = \"SiteTree\".\"ID\"\n\nWHERE (\"VirtualPage\".\"CopyContentFromID\" = ?)\n AND (\"SiteTree\".\"ClassName\" IN (?))\n\nORDER BY \"SiteTree\".\"Sort\" ASC\n\nTable 'SS_mysite.Page' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-21 09:49:25] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 09:49:44] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 10:07:38] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:08:37.758903 2022] [mpm_prefork:notice] [pid 841] AH00173: SIGHUP received.  Attempting to restart
[Thu Apr 21 10:08:37.788501 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:08:37.788517 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:09:12] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:11:55.566104 2022] [mpm_prefork:notice] [pid 841] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 10:11:55.588873 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:11:55.588884 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:12:02] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 10:12:13] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:16:31.355582 2022] [mpm_prefork:notice] [pid 841] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 10:16:31.384285 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:16:31.384306 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:18:38] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 10:19:33] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:22:54.439883 2022] [mpm_prefork:notice] [pid 841] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 10:22:54.461783 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:22:54.461792 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:22:59] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:23:57.165763 2022] [mpm_prefork:notice] [pid 841] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 10:23:57.187304 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:23:57.187313 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:24:02] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 10:24:09] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 10:30:09] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:35:19.975365 2022] [mpm_prefork:notice] [pid 841] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 10:35:20.000169 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:35:20.000180 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:36:07] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:37:09.355435 2022] [mpm_prefork:notice] [pid 841] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 10:37:09.384972 2022] [mpm_prefork:notice] [pid 841] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:37:09.384985 2022] [core:notice] [pid 841] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:37:13] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:38:41.929326 2022] [mpm_prefork:notice] [pid 862] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:38:41.929358 2022] [core:notice] [pid 862] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:38:55] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 10:47:53.469377 2022] [mpm_prefork:notice] [pid 846] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 10:47:53.469407 2022] [core:notice] [pid 846] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 10:48:07] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 14:53:37.843832 2022] [mpm_prefork:notice] [pid 809] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 14:53:37.843939 2022] [core:notice] [pid 809] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 14:54:01] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 15:15:15.740557 2022] [mpm_prefork:notice] [pid 813] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 15:15:15.740586 2022] [core:notice] [pid 813] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 15:38:00] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 15:41:11] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Thu Apr 21 15:41:59.709590 2022] [mpm_prefork:notice] [pid 813] AH00171: Graceful restart requested, doing restart
[Thu Apr 21 15:41:59.755351 2022] [mpm_prefork:notice] [pid 813] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 15:41:59.755365 2022] [core:notice] [pid 813] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 15:43:18] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 15:44:35] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 15:45:02] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 15:50:03] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 16:02:25] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 16:03:43] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 16:07:14] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 16:08:37] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 77" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 77 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 16:13:52] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 90" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 90 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 17:06:23] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" - 1, "LastResponseTimestamp" = ?  WHERE (ID = ?)  Table 'ss_tmpdb_1650517480_4312105.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" - 1, \"LastResponseTimestamp\" = ?\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650517480_4312105.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[Thu Apr 21 17:07:32.337386 2022] [mpm_prefork:notice] [pid 795] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 21 17:07:32.337421 2022] [core:notice] [pid 795] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-21 17:09:34] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 17:16:49] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 17:17:42] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:17:51] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:18:12] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:19:16] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:19:25] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:19:46] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:34:57] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:35:06] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:35:26] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:36:04] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:36:14] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:36:41] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "database" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php line 281 {"exception":"[object] (Error(code: 0): Attempt to assign property \"database\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php:281)"} []
[2022-04-21 17:36:42] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:37:35] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:37:45] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:38:15] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:43:03] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-21 17:44:01] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:44:10] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:44:31] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:45:20] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:45:29] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:45:50] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:46:38] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" + 1  WHERE (ID = ?)  No database selected" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" + 1\n\nWHERE (ID = ?)\n\nNo database selected at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-21 17:46:57] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:47:07] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[2022-04-21 17:47:27] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\Types::Block()" at /var/www/.graphql/admin/Query.php line 95 {"exception":"[object] (Error(code: 0): Call to undefined method SSGraphQLSchema_21232f297a57a5a743894a0e4a801fc3\\Types::Block() at /var/www/.graphql/admin/Query.php:95)"} []
[Fri Apr 22 09:40:23.603446 2022] [mpm_prefork:notice] [pid 805] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Fri Apr 22 09:40:23.603905 2022] [core:notice] [pid 805] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-22 09:49:59] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" - 1, "LastResponseTimestamp" = ?  WHERE (ID = ?)  Table 'ss_tmpdb_1650577687_1245583.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" - 1, \"LastResponseTimestamp\" = ?\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650577687_1245583.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-22 09:51:39] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" + 1  WHERE (ID = ?)  Table 'ss_tmpdb_1650577810_4816477.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" + 1\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650577810_4816477.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-22 09:51:42] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" + 1  WHERE (ID = ?)  Table 'ss_tmpdb_1650577810_4816477.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" + 1\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650577810_4816477.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-22 09:53:41] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" - 1, "LastResponseTimestamp" = ?  WHERE (ID = ?)  Table 'ss_tmpdb_1650577913_3675040.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" - 1, \"LastResponseTimestamp\" = ?\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650577913_3675040.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64, SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nINSERT INTO \"SiteConfig\"\n (\"ClassName\", \"Title\", \"Tagline\", \"CanViewType\", \"CanEditType\", \"CanCreateTopLevelType\", \"LastEdited\", \"Created\")\n VALUES\n (?, ?, ?, ?, ?, ?, ?, ?)\n\nTable 'ss_tmpdb_1650577913_3675040.SiteConfig' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-22 09:56:13] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" - 1, "LastResponseTimestamp" = ?  WHERE (ID = ?)  Table 'ss_tmpdb_1650578062_2315300.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" - 1, \"LastResponseTimestamp\" = ?\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650578062_2315300.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-22 10:19:47] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 10:22:52] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 10:31:49] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  SELECT DISTINCT "SiteConfig"."ClassName", "SiteConfig"."LastEdited", "SiteConfig"."Created", "SiteConfig"."Title", "SiteConfig"."Tagline", "SiteConfig"."CanViewType", "SiteConfig"."CanEditType", "SiteConfig"."CanCreateTopLevelType", "SiteConfig"."ID",  			CASE WHEN "SiteConfig"."ClassName" IS NOT NULL THEN "SiteConfig"."ClassName" 			ELSE 'SilverStripe\\SiteConfig\\SiteConfig' END AS "RecordClassName"  FROM "SiteConfig"  LIMIT 1  Table 'SS_mysite.SiteConfig' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nSELECT DISTINCT \"SiteConfig\".\"ClassName\", \"SiteConfig\".\"LastEdited\", \"SiteConfig\".\"Created\", \"SiteConfig\".\"Title\", \"SiteConfig\".\"Tagline\", \"SiteConfig\".\"CanViewType\", \"SiteConfig\".\"CanEditType\", \"SiteConfig\".\"CanCreateTopLevelType\", \"SiteConfig\".\"ID\", \n\t\t\tCASE WHEN \"SiteConfig\".\"ClassName\" IS NOT NULL THEN \"SiteConfig\".\"ClassName\"\n\t\t\tELSE 'SilverStripe\\\\SiteConfig\\\\SiteConfig' END AS \"RecordClassName\"\n\nFROM \"SiteConfig\"\n\nLIMIT 1\n\nTable 'SS_mysite.SiteConfig' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-22 10:47:02] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[Fri Apr 22 10:49:38.866003 2022] [mpm_prefork:notice] [pid 805] AH00171: Graceful restart requested, doing restart
[Fri Apr 22 10:49:38.914067 2022] [mpm_prefork:notice] [pid 805] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Fri Apr 22 10:49:38.914079 2022] [core:notice] [pid 805] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-22 10:50:27] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 11:01:18] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 11:05:47] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 11:18:34] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 11:32:08] error-log.ERROR: Uncaught Exception TypeError: "GraphQL\Type\Definition\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\Type\Definition\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75" at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php line 294 {"exception":"[object] (TypeError(code: 0): GraphQL\\Type\\Definition\\Type::getNullableType(): Argument #1 ($type) must be of type GraphQL\\Type\\Definition\\Type, null given, called in /var/www/vendor/webonyx/graphql-php/src/Validator/Rules/ValuesOfCorrectType.php on line 75 at /var/www/vendor/webonyx/graphql-php/src/Type/Definition/Type.php:294)"} []
[2022-04-22 14:10:31] error-log.WARNING: E_WARNING: preg_replace(): Empty regular expression {"code":2,"message":"preg_replace(): Empty regular expression","file":"/var/www/app/_config.php","line":11} []
[2022-04-22 14:12:42] error-log.WARNING: E_WARNING: preg_match(): Empty regular expression {"code":2,"message":"preg_match(): Empty regular expression","file":"/var/www/app/_config.php","line":11} []
[2022-04-22 14:29:35] error-log.ERROR: Uncaught Exception ValueError: "Path cannot be empty" at /var/www/app/_config.php line 11 {"exception":"[object] (ValueError(code: 0): Path cannot be empty at /var/www/app/_config.php:11)"} []
[2022-04-22 14:29:59] error-log.ERROR: Uncaught Exception ValueError: "Path cannot be empty" at /var/www/app/_config.php line 11 {"exception":"[object] (ValueError(code: 0): Path cannot be empty at /var/www/app/_config.php:11)"} []
[2022-04-22 14:33:10] error-log.ERROR: Uncaught Exception ValueError: "Path cannot be empty" at /var/www/app/_config.php line 11 {"exception":"[object] (ValueError(code: 0): Path cannot be empty at /var/www/app/_config.php:11)"} []
[2022-04-22 14:59:20] error-log.WARNING: E_WARNING: unlink(): No such file or directory {"code":2,"message":"unlink(): No such file or directory","file":"/var/www/app/_config.php","line":11} []
[Sat Apr 23 00:16:00.006814 2022] [mpm_prefork:notice] [pid 817] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Sat Apr 23 00:16:00.007430 2022] [core:notice] [pid 817] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-23 00:26:40] error-log.ERROR: Uncaught Exception Error: "Call to a member function getConfig() on null" at /var/www/vendor/silverstripe/admin/code/SecurityAdmin.php line 209 {"exception":"[object] (Error(code: 0): Call to a member function getConfig() on null at /var/www/vendor/silverstripe/admin/code/SecurityAdmin.php:209)"} []
[2022-04-23 00:27:13] error-log.ERROR: Uncaught Exception Error: "Call to a member function getConfig() on null" at /var/www/vendor/silverstripe/admin/code/SecurityAdmin.php line 209 {"exception":"[object] (Error(code: 0): Call to a member function getConfig() on null at /var/www/vendor/silverstripe/admin/code/SecurityAdmin.php:209)"} []
[Tue Apr 26 17:04:45.626182 2022] [mpm_prefork:notice] [pid 783] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Tue Apr 26 17:04:45.626925 2022] [core:notice] [pid 783] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-26 17:50:20] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" - 1, "LastResponseTimestamp" = ?  WHERE (ID = ?)  Table 'ss_tmpdb_1650951995_1144830.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" - 1, \"LastResponseTimestamp\" = ?\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650951995_1144830.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-26 17:54:34] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  UPDATE "TestSessionState"  SET "PendingRequests" = "PendingRequests" - 1, "LastResponseTimestamp" = ?  WHERE (ID = ?)  Table 'ss_tmpdb_1650952231_7252854.TestSessionState' doesn't exist" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nUPDATE \"TestSessionState\"\n SET \"PendingRequests\" = \"PendingRequests\" - 1, \"LastResponseTimestamp\" = ?\n\nWHERE (ID = ?)\n\nTable 'ss_tmpdb_1650952231_7252854.TestSessionState' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64, SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nSELECT DISTINCT \"SiteConfig\".\"ClassName\", \"SiteConfig\".\"LastEdited\", \"SiteConfig\".\"Created\", \"SiteConfig\".\"Title\", \"SiteConfig\".\"Tagline\", \"SiteConfig\".\"CanViewType\", \"SiteConfig\".\"CanEditType\", \"SiteConfig\".\"CanCreateTopLevelType\", \"SiteConfig\".\"ID\", \n\t\t\tCASE WHEN \"SiteConfig\".\"ClassName\" IS NOT NULL THEN \"SiteConfig\".\"ClassName\"\n\t\t\tELSE 'SilverStripe\\\\SiteConfig\\\\SiteConfig' END AS \"RecordClassName\"\n\nFROM \"SiteConfig\"\n\nLIMIT 1\n\nTable 'ss_tmpdb_1650952231_7252854.SiteConfig' doesn't exist at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[Wed Apr 27 09:41:44.500891 2022] [mpm_prefork:notice] [pid 802] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 27 09:41:44.501555 2022] [core:notice] [pid 802] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-27 09:50:56] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:00] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:05] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:09] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:14] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:18] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:22] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:27] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:30] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:34] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:51:58] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[2022-04-27 09:52:29] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "session" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php line 95 {"exception":"[object] (Error(code: 0): Attempt to assign property \"session\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionHTTPMiddleware.php:95)"} []
[Wed Apr 27 11:30:42.062374 2022] [mpm_prefork:notice] [pid 802] AH00171: Graceful restart requested, doing restart
[Wed Apr 27 11:30:42.139567 2022] [mpm_prefork:notice] [pid 802] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 27 11:30:42.139586 2022] [core:notice] [pid 802] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-27 11:46:33] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "database" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php line 281 {"exception":"[object] (Error(code: 0): Attempt to assign property \"database\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php:281)"} []
[2022-04-27 11:53:25] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "database" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php line 281 {"exception":"[object] (Error(code: 0): Attempt to assign property \"database\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php:281)"} []
[2022-04-27 11:54:14] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "database" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php line 281 {"exception":"[object] (Error(code: 0): Attempt to assign property \"database\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php:281)"} []
[Wed Apr 27 14:07:45.240916 2022] [mpm_prefork:notice] [pid 806] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 27 14:07:45.241351 2022] [core:notice] [pid 806] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Wed Apr 27 14:11:05.749671 2022] [mpm_prefork:notice] [pid 806] AH00171: Graceful restart requested, doing restart
[Wed Apr 27 14:11:05.789791 2022] [mpm_prefork:notice] [pid 806] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Wed Apr 27 14:11:05.789813 2022] [core:notice] [pid 806] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-27 14:11:14] error-log.ERROR: Uncaught Exception TypeError: "get_parent_class(): Argument #1 ($object_or_class) must be an object or a valid class name, string given" at /var/www/vendor/silverstripe/framework/src/ORM/DataObjectSchema.php line 148 {"exception":"[object] (TypeError(code: 0): get_parent_class(): Argument #1 ($object_or_class) must be an object or a valid class name, string given at /var/www/vendor/silverstripe/framework/src/ORM/DataObjectSchema.php:148)"} []
[2022-04-27 14:12:58] error-log.ERROR: Uncaught Exception InvalidArgumentException: "Page references nonexistent DNADesign\Elemental\Extensions\ElementalPageExtension in 'extensions'" at /var/www/vendor/silverstripe/framework/src/Core/Config/Middleware/ExtensionMiddleware.php line 70 {"exception":"[object] (InvalidArgumentException(code: 0): Page references nonexistent DNADesign\\Elemental\\Extensions\\ElementalPageExtension in 'extensions' at /var/www/vendor/silverstripe/framework/src/Core/Config/Middleware/ExtensionMiddleware.php:70)"} []
[2022-04-27 15:22:31] error-log.ERROR: Uncaught Exception InvalidArgumentException: "SilverStripe\ORM\FieldType\DBField references nonexistent SilverStripe\GraphQL\Scaffolding\Extensions\TypeCreatorExtension in 'extensions'" at /var/www/vendor/silverstripe/framework/src/Core/Config/Middleware/ExtensionMiddleware.php line 70 {"exception":"[object] (InvalidArgumentException(code: 0): SilverStripe\\ORM\\FieldType\\DBField references nonexistent SilverStripe\\GraphQL\\Scaffolding\\Extensions\\TypeCreatorExtension in 'extensions' at /var/www/vendor/silverstripe/framework/src/Core/Config/Middleware/ExtensionMiddleware.php:70)"} []
[2022-04-27 18:44:59] error-log.ERROR: Uncaught Exception TypeError: "SilverStripe\SessionManager\Models\LoginSession::generate(): Argument #1 ($member) must be of type SilverStripe\Security\Member, null given, called in /var/www/app/_config.php on line 19" at /var/www/vendor/silverstripe/session-manager/src/Models/LoginSession.php line 210 {"exception":"[object] (TypeError(code: 0): SilverStripe\\SessionManager\\Models\\LoginSession::generate(): Argument #1 ($member) must be of type SilverStripe\\Security\\Member, null given, called in /var/www/app/_config.php on line 19 at /var/www/vendor/silverstripe/session-manager/src/Models/LoginSession.php:210)"} []
[2022-04-27 19:14:34] error-log.ERROR: Uncaught Exception Error: "Attempt to assign property "database" on null" at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php line 281 {"exception":"[object] (Error(code: 0): Attempt to assign property \"database\" on null at /var/www/vendor/silverstripe/testsession/src/TestSessionEnvironment.php:281)"} []
[Thu Apr 28 09:38:50.313603 2022] [mpm_prefork:notice] [pid 791] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 28 09:38:50.314263 2022] [core:notice] [pid 791] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[Thu Apr 28 11:16:52.601780 2022] [mpm_prefork:notice] [pid 791] AH00171: Graceful restart requested, doing restart
[Thu Apr 28 11:16:52.749223 2022] [mpm_prefork:notice] [pid 791] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Thu Apr 28 11:16:52.749232 2022] [core:notice] [pid 791] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-28 15:44:27] error-log.ERROR: Uncaught Exception TypeError: "array_keys(): Argument #1 ($array) must be of type array, null given" at /var/www/vendor/silverstripe/framework/src/Control/Session.php line 458 {"exception":"[object] (TypeError(code: 0): array_keys(): Argument #1 ($array) must be of type array, null given at /var/www/vendor/silverstripe/framework/src/Control/Session.php:458)"} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:55:40] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 15:57:26] error-log.WARNING: E_WARNING: file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory {"code":2,"message":"file_put_contents(/var/www/4-10_1/www/artifacts/temp.txt): Failed to open stream: No such file or directory","file":"/var/www/app/_config.php","line":29} []
[2022-04-28 16:39:47] error-log.ERROR: Uncaught Exception Error: "Class "SilverStripe\HybridSessions\HybridSession" not found" at /var/www/app/_config.php line 25 {"exception":"[object] (Error(code: 0): Class \"SilverStripe\\HybridSessions\\HybridSession\" not found at /var/www/app/_config.php:25)"} []
[2022-04-28 16:39:49] error-log.ERROR: Uncaught Exception Error: "Class "SilverStripe\HybridSessions\HybridSession" not found" at /var/www/app/_config.php line 25 {"exception":"[object] (Error(code: 0): Class \"SilverStripe\\HybridSessions\\HybridSession\" not found at /var/www/app/_config.php:25)"} []
[2022-04-28 16:39:50] error-log.ERROR: Uncaught Exception Error: "Class "SilverStripe\HybridSessions\HybridSession" not found" at /var/www/app/_config.php line 25 {"exception":"[object] (Error(code: 0): Class \"SilverStripe\\HybridSessions\\HybridSession\" not found at /var/www/app/_config.php:25)"} []
[2022-04-28 16:39:52] error-log.ERROR: Uncaught Exception Error: "Class "SilverStripe\HybridSessions\HybridSession" not found" at /var/www/app/_config.php line 25 {"exception":"[object] (Error(code: 0): Class \"SilverStripe\\HybridSessions\\HybridSession\" not found at /var/www/app/_config.php:25)"} []
[Fri Apr 29 09:36:08.879587 2022] [mpm_prefork:notice] [pid 250] AH00163: Apache/2.4.53 (Ubuntu) configured -- resuming normal operations
[Fri Apr 29 09:36:08.880394 2022] [core:notice] [pid 250] AH00094: Command line: '/usr/sbin/apache2 -D FOREGROUND'
[2022-04-29 11:22:24] error-log.ERROR: Uncaught Exception BadMethodCallException: "Object->__call(): the method 'getParentRelationship' does not exist on 'SilverStripe\ORM\ManyManyThroughList'" at /var/www/vendor/silverstripe/framework/src/Core/CustomMethods.php line 54 {"exception":"[object] (BadMethodCallException(code: 0): Object->__call(): the method 'getParentRelationship' does not exist on 'SilverStripe\\ORM\\ManyManyThroughList' at /var/www/vendor/silverstripe/framework/src/Core/CustomMethods.php:54)"} []
[2022-04-29 11:55:50] error-log.ERROR: Uncaught Exception SilverStripe\ORM\Connect\DatabaseException: "Couldn't run query:  DELETE  FROM "TeamSupporter"  WHERE ("TeamSupporter"."" IN  SELECT "Supporter"."ID"  FROM "Supporter"))  Unknown column 'TeamSupporter.' in 'IN/ALL/ANY subquery'" at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php line 64 {"exception":"[object] (SilverStripe\\ORM\\Connect\\DatabaseException(code: 0): Couldn't run query:\n\nDELETE\n\nFROM \"TeamSupporter\"\n\nWHERE (\"TeamSupporter\".\"\" IN \nSELECT \"Supporter\".\"ID\"\n\nFROM \"Supporter\"))\n\nUnknown column 'TeamSupporter.' in 'IN/ALL/ANY subquery' at /var/www/vendor/silverstripe/framework/src/ORM/Connect/DBConnector.php:64)"} []
[2022-04-29 12:41:29] error-log.ERROR: Uncaught Exception Error: "Call to undefined method SilverStripe\ORM\Connect\MySQLStatement::count()" at /var/www/vendor/silverstripe/framework/src/ORM/ManyManyThroughList.php line 199 {"exception":"[object] (Error(code: 0): Call to undefined method SilverStripe\\ORM\\Connect\\MySQLStatement::count() at /var/www/vendor/silverstripe/framework/src/ORM/ManyManyThroughList.php:199)"} []
[2022-04-29 12:46:53] error-log.WARNING: E_WARNING: Undefined variable $team {"code":2,"message":"Undefined variable $team","file":"/var/www/app/_config.php","line":21} []
[2022-04-29 12:46:53] error-log.ERROR: Uncaught Exception Error: "Call to a member function SupportNormal() on null" at /var/www/app/_config.php line 21 {"exception":"[object] (Error(code: 0): Call to a member function SupportNormal() on null at /var/www/app/_config.php:21)"} []
[2022-04-29 13:08:20] error-log.ERROR: Uncaught Exception InvalidArgumentException: "Couldn't find "TeamSupporter"."TeamID" in the query filter." at /var/www/vendor/silverstripe/framework/src/ORM/DataQuery.php line 156 {"exception":"[object] (InvalidArgumentException(code: 0): Couldn't find \"TeamSupporter\".\"TeamID\" in the query filter. at /var/www/vendor/silverstripe/framework/src/ORM/DataQuery.php:156)"} []
[2022-04-29 13:08:51] error-log.ERROR: Uncaught Exception InvalidArgumentException: "Couldn't find "TeamSupporter"."TeamID" in the query filter." at /var/www/vendor/silverstripe/framework/src/ORM/DataQuery.php line 156 {"exception":"[object] (InvalidArgumentException(code: 0): Couldn't find \"TeamSupporter\".\"TeamID\" in the query filter. at /var/www/vendor/silverstripe/framework/src/ORM/DataQuery.php:156)"} []
[2022-04-29 13:12:06] error-log.ERROR: Uncaught Exception InvalidArgumentException: "Couldn't find "TeamSupporter"."TeamID" in the query filter." at /var/www/vendor/silverstripe/framework/src/ORM/DataQuery.php line 156 {"exception":"[object] (InvalidArgumentException(code: 0): Couldn't find \"TeamSupporter\".\"TeamID\" in the query filter. at /var/www/vendor/silverstripe/framework/src/ORM/DataQuery.php:156)"} []
[2022-04-29 14:50:32] error-log.WARNING: E_WARNING: foreach() argument must be of type array|object, null given {"code":2,"message":"foreach() argument must be of type array|object, null given","file":"/var/www/vendor/dnadesign/silverstripe-populate/code/Populate.php","line":84} []
[2022-04-29 14:50:32] error-log.WARNING: E_USER_WARNING: Couldn't set response type to 500 because of output on line 419 of /var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php {"code":512,"message":"Couldn't set response type to 500 because of output on line 419 of /var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php","file":"/var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php","line":397} []
[2022-04-29 14:52:26] error-log.WARNING: E_WARNING: foreach() argument must be of type array|object, null given {"code":2,"message":"foreach() argument must be of type array|object, null given","file":"/var/www/vendor/dnadesign/silverstripe-populate/code/Populate.php","line":84} []
[2022-04-29 14:52:26] error-log.WARNING: E_USER_WARNING: Couldn't set response type to 500 because of output on line 419 of /var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php {"code":512,"message":"Couldn't set response type to 500 because of output on line 419 of /var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php","file":"/var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php","line":397} []
[2022-04-29 14:53:46] error-log.WARNING: E_WARNING: foreach() argument must be of type array|object, null given {"code":2,"message":"foreach() argument must be of type array|object, null given","file":"/var/www/vendor/dnadesign/silverstripe-populate/code/Populate.php","line":84} []
[2022-04-29 14:53:46] error-log.WARNING: E_USER_WARNING: Couldn't set response type to 500 because of output on line 419 of /var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php {"code":512,"message":"Couldn't set response type to 500 because of output on line 419 of /var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php","file":"/var/www/vendor/silverstripe/framework/src/Control/HTTPResponse.php","line":397} []
